test/signs_ui/signs/state_test.exs

```diff
@@ -1,11 +1,23 @@
 defmodule SignsUI.Signs.StateTest do
-  use ExUnit.Case
-  use Phoenix.ChannelTest
+  use SignsUiWeb.ChannelCase
   import SignsUI.Signs.State
   alias SignsUI.Signs
   alias SignsUI.Signs.Sign
 
-  @endpoint SignsUiWeb.Endpoint
+  setup _tags do
```

Comment:
If you aren't using the tags, you can do `setup do`

Reply:
Done